### PR TITLE
Create the file path on the tool service

### PR DESCRIPTION
**Pull request by aaomidi**
We need to create this path on the tooling side instead of ADS.

Fixes --- https://github.com/microsoft/azuredatastudio/pull/7523

### CODE REVIEW

File: src/Microsoft.SqlTools.ServiceLayer/Scripting/ScriptingService.cs [hunk 1]
```diff
@@ -126,6 +127,11 @@ public async Task HandleScriptExecuteRequest(ScriptingParams parameters, Request
                     }
                 }
 
+                if (parameters.FilePath == null)
+                {
+                    parameters.FilePath = Path.Join(Path.GetTempPath(), $"{Convert.ToBase64String(Guid.NewGuid().ToByteArray()).Substring(0, 8)}-{DateTime.Now:yyyy-MM-dd_hh-mm-ss-fff}");
```
Comment (llali) [inline src/Microsoft.SqlTools.ServiceLayer/Scripting/ScriptingService.cs]: please add a comment to explain what the path would look like and reason for it.

Comment (aaomidi) [reply src/Microsoft.SqlTools.ServiceLayer/Scripting/ScriptingService.cs]: A comment on the code or here?

Comment (llali) [inline src/Microsoft.SqlTools.ServiceLayer/Scripting/ScriptingService.cs]: Also please explain in the PR how this is going to fix the original issue

Comment (llali) [reply src/Microsoft.SqlTools.ServiceLayer/Scripting/ScriptingService.cs]: in the code. it's useful to have a sample in a comment, makes it more clear. also may add some comment about why you did that. trying to create a random file name?

**Comment by aaomidi** [comment]
We're currently generating the file path for this operation on the frontend of ADS. This is causing issues for our exploration work and we need to generate it either on the mssql extension on ADS or the tools service. 

Moving this here allows me to get rid of the import 'os'; on the area of the app where it shouldn't have one.

File: src/Microsoft.SqlTools.ServiceLayer/Scripting/ScriptingService.cs (hunk 1 above)
Comment (kburtram) [reply src/Microsoft.SqlTools.ServiceLayer/Scripting/ScriptingService.cs]: @aaomidi any reason to not use [GetTempFileName](https://docs.microsoft.com/en-us/dotnet/api/system.io.path.gettempfilename?view=netcore-3.0)?

Comment (kburtram) [reply src/Microsoft.SqlTools.ServiceLayer/Scripting/ScriptingService.cs]: @aaomidi double checking the docs, it looks like [GetRandomFileName](https://docs.microsoft.com/en-us/dotnet/api/system.io.path.getrandomfilename?view=netcore-3.0) is the one I'm asking about.  GetTempFileName appears to actually create a file, which isn't what we want I think.

**Comment by aaomidi** [comment]
Nope! I just don't know C# as well as I should @kburtram

**Pull request merged**